{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py

```diff
@@ -1 +1 @@
-
+default_app_config = 'home.apps.MainAppConfig'
```

Comment:
I think we don't want to use this - 
https://docs.djangoproject.com/en/1.9/ref/applications/#configuring-applications

New applications should avoid default_app_config. Instead they should require the dotted path to the appropriate AppConfig subclass to be configured explicitly in INSTALLED_APPS.

https://docs.djangoproject.com/en/1.9/ref/applications/#configuring-applications